### PR TITLE
Resolves #2759

### LINKED ISSUE #2759: Get-ADUser - Error in description - "name" is not supported for identity
Hello,

i was just trying if there are any possibilities to match multiple, or at least the wrong, users when using get-aduser with identity. Regarding the "description" part at the top you might get wrong users as you could get "samaccountname" or "name" matching. Reading the **section for the parameter "identity" itself, there is no "name" attribute mentioned**.
Testing showed that doing a get-aduser for a user with it's "name" attribute does not seem to work.

Here is the wrong sentence i meant:
> The Identity parameter specifies the Active Directory user to get. You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, **or name**. 

Just a side note - In Microsoft Exchange the get-mailbox -identity parameter can search with "name" attribute and some more. That caused some of my scripts to match the wrong user so be aware of that. That's why i thougth get-aduser might have some of these "problems" either.

thanks
An-dir

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 42d0073f-ab96-bd7f-f838-c931dd5e9cb8
* Version Independent ID: b013a1e2-5879-5924-9dbd-a8db54885d10
* Content: [Get-ADUser (ActiveDirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/get-aduser?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/activedirectory/Get-ADUser.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/activedirectory/Get-ADUser.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

### PR DESCRIPTION
**Pull request by velkovb**
Correct description.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2759

### CODE REVIEW

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Get-ADUser [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <St
 The **Get-ADUser** cmdlet gets a specified user object or performs a search to get multiple user objects.
 
 The *Identity* parameter specifies the Active Directory user to get.
-You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, or name.
+You can identify a user by its distinguished name (DN), GUID, security identifier (SID) or Security Account Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:39]:
```suggestion
You can identify a user by its distinguished name (DN), GUID, security identifier (SID), or Security Account Manager (SAM) account name.
```

File: docset/winserver2012r2-ps/activedirectory/Get-ADUser.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Get-ADUser [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <St
 The **Get-ADUser** cmdlet gets a specified user object or performs a search to get multiple user objects.
 
 The *Identity* parameter specifies the Active Directory user to get.
-You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, or name.
+You can identify a user by its distinguished name (DN), GUID, security identifier (SID) or Security Account Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/activedirectory/Get-ADUser.md:41]:
```suggestion
You can identify a user by its distinguished name (DN), GUID, security identifier (SID), or Security Account Manager (SAM) account name.
```

File: docset/winserver2016-ps/activedirectory/Get-ADUser.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Get-ADUser [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <St
 The **Get-ADUser** cmdlet gets a specified user object or performs a search to get multiple user objects.
 
 The *Identity* parameter specifies the Active Directory user to get.
-You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, or name.
+You can identify a user by its distinguished name (DN), GUID, security identifier (SID) or Security Account Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/activedirectory/Get-ADUser.md:42]:
```suggestion
You can identify a user by its distinguished name (DN), GUID, security identifier (SID), or Security Account Manager (SAM) account name.
```

File: docset/winserver2019-ps/activedirectory/Get-ADUser.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ Get-ADUser [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <St
 The **Get-ADUser** cmdlet gets a specified user object or performs a search to get multiple user objects.
 
 The *Identity* parameter specifies the Active Directory user to get.
-You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, or name.
+You can identify a user by its distinguished name (DN), GUID, security identifier (SID) or Security Account Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:42]:
```suggestion
You can identify a user by its distinguished name (DN), GUID, security identifier (SID), or Security Account Manager (SAM) account name.
```

File: docset/winserver2022-ps/activedirectory/Get-ADUser.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ Get-ADUser [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <St
 The **Get-ADUser** cmdlet gets a specified user object or performs a search to get multiple user objects.
 
 The *Identity* parameter specifies the Active Directory user to get.
-You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name, or name.
+You can identify a user by its distinguished name (DN), GUID, security identifier (SID), Security Account Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/Get-ADUser.md:42]:
```suggestion
You can identify a user by its distinguished name (DN), GUID, security identifier (SID), or Security Account Manager (SAM) account name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please re-insert the Oxford comma.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**